lib/exfile/processor/content_type.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Exfile.Processor.ContentType do
+  @behaviour Exfile.Processor
+
+  def call(file = %{ meta: %{ "content_type" => content_type }}, _, _) when is_binary(content_type) do
+    { :ok, file }
+  end
+
+  def call(file, _, _) do
+    file = Exfile.ProcessorChain.coerce_to_local_file(file)
```

Comment:
I found error here!

We are getting `%Exfile.File{}` struct here, but putting `%Exfile.Localfile{}` to changeset back! This is the reason why things like https://github.com/keichan34/exfile/blob/master/lib/exfile/phoenix/helpers.ex#L76 can't pattern match